File: src/main/java/br/com/leonardoferreira/jirareport/domain/vo/changelog/JiraChangelogAuthor.java
package br.com.leonardoferreira.jirareport.domain.vo.changelog;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

/**
 * @author s2it_leferreira
 * @since 11/06/18 13:43
 */
@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class JiraChangelogAuthor {

    private String self;

    @JsonProperty("name")
    private String username;

    @JsonProperty("key")
    private String key;

    @JsonProperty("emailAddress")
    private String email;

    @JsonProperty("displayName")
    private String name;

    private Boolean active;

    private String timeZone;
}

File: src/main/java/br/com/jiratorio/domain/changelog/JiraChangelogAuthor.kt
package br.com.jiratorio.domain.changelog

import com.fasterxml.jackson.annotation.JsonProperty

data class JiraChangelogAuthor(
    var self: String? = null,

    @JsonProperty("name")
    var username: String? = null,

    @JsonProperty("key")
    var key: String? = null,

    @JsonProperty("emailAddress")
    var email: String? = null,

    @JsonProperty("displayName")
    var name: String? = null,

    var active: Boolean? = null,

    var timeZone: String? = null
)
